File: LeetCode/剑指Offer/03.py
class Solution:
    def findRepeatNumber(self, nums: List[int]) -> int:
        a = [0] * len(nums)
        for i in range(len(nums)):
            if(a[nums[i]] == 0):
                a[nums[i]]+=1
            else:
                ans = nums[i]
                break
        return ans

File: LeetCode/剑指Offer/03.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findRepeatNumber = function(nums) {
    var a=[];
    for(var i=0; i < nums.length; i++){
        a[i]=0;
    }
    for(var i=0; i < nums.length; i++){
        if(a[nums[i]]===0){
            a[nums[i]]+=1;
        }else{
            return nums[i];
        }
    }
};
